packages/cozy-mespapiers-lib/src/components/Search/search.js

```diff
@@ -18,11 +20,20 @@ export const index = new Document({
     tag: 'flexsearchProps:tag',
     index: [
       'name', // io.cozy.files, io.cozy.contacts
+      'metadata:ibanNumber', // io.cozy.files
+      'metadata:number', // io.cozy.files
+      'metadata:AObtentionDate', // io.cozy.files
+      'metadata:BObtentionDate', // io.cozy.files
+      'metadata:CObtentionDate', // io.cozy.files
+      'metadata:DObtentionDate', // io.cozy.files
```

Comment:
Sync required with @BenjaminMty, those date attributes should be removed

Reply:
oui je sais, sync faite, prévue dans une autre PR